universal/ubi9/Dockerfile

```diff
@@ -413,18 +414,12 @@ EOF
 RUN echo 'export SDKMAN_DIR="/home/tooling/.sdkman"' >> ${PROFILE_EXT}
 RUN echo '[[ -s "$SDKMAN_DIR/bin/sdkman-init.sh" ]] && source "$SDKMAN_DIR/bin/sdkman-init.sh"' >> ${PROFILE_EXT}
 
-
-# Create symbolic links from /home/tooling/ -> /home/user/
-RUN stow . -t /home/user/ -d /home/tooling/ --no-folding
-
```

Comment:
To be on the safer side, instead of removing the stow call, could we instead add `.config` in the stow ignore file [here](https://github.com/devfile/developer-images/blob/main/base/ubi9/.stow-local-ignore)?

We have the stow call here to move certain /home/tooling folders ([example](https://github.com/devfile/developer-images/blob/e2f4db7381e20438f6f30b7c8dd3dfbb62c366ce/universal/ubi9/Dockerfile#L77-L78)) to /home/user.

The stow call might not be entirely necessary since the binaries installed under /home/tooling are being added to the PATH anyways. I'll have to investigate more to be sure whether the stow call can be removed.

Reply:
Do we need the `stow` call during build?  I don't think that it does anything useful that isn't done when it runs in the `entrypoint.sh`

You'll see that exact same syntax is run in `entrypoint.sh`.  When it runs there, it runs as the user not root.

Reply:
@dkwon17 I restored the `stow` command in the Dockerfile.